x-pack/test/functional/apps/monitoring/kibana/instances_mb.js

```diff
@@ -23,6 +23,7 @@ export default function ({ getService, getPageObjects }) {
         {
           from: 'Aug 29, 2017 @ 17:24:14.254',
           to: 'Aug 29, 2017 @ 17:25:44.142',
+          useCreate: true,
```

Comment:
Out of scope: This is set since we're writing to a data stream right? I wonder if perhaps it would be nicer if our setup methods used that domain language instead of "useCreate" which is a detail of how es_archiver decides to put data into ES if it's a data stream?

It's very unclear to me from just reading this call site what this parameter does.

Reply:
Agreed, I'll update that as part of the [logstash PR](https://github.com/elastic/kibana/pull/122177)